main.py

```diff
@@ -0,0 +1,37 @@
+#importing the necessary libraries
```

Comment:
Comments should generally only be used where _absolutely_ necessary. Where the code is self explanatory, or can be easily understood, there's no need to use them 😄